README.md

```diff
@@ -620,12 +620,12 @@ docker run \
 
 **Notes:**
 
-* The volume `/var/run/libvirt/libvirt-sock` is mandatory if you want to run Terrible locally (a local QEMU/KVM instance). In this way you directly interact with the QEMU/KVM api, provided by the system.
+* The volume `/var/run/libvirt/libvirt-sock` is mandatory if you want to run Terrible locally (a local QEMU/KVM instance). In this way you directly interact with the QEMU/KVM api, provided by the system (required).
 
-* The volume `./inventory-test.yml` is to include the inventory inside the container, to deploy the infrastructure.
+* The volume `./inventory-test.yml` is to include the inventory inside the container, to deploy the infrastructure (optional).
 
-* The volume `~/VirtualMachines/` is to include the `qcow2` images inside the container to deploy them.
-* The volume `~/.ssh` is to include your ssh keys into the container and deploy them inside the infrastructure machines.
+* The volume `~/path/to/vm/images/` is to include the `qcow2` images (previously generated with Packer) inside the container to deploy them (required).
+* The volume `~/.ssh` is to include your ssh keys into the container and deploy them inside the infrastructure machines (optional).
```

Comment:
Required if an ssh_key is declared in the inventory

Reply:
you can generate ssh keys inside container for a one-time use

Reply:
Not right now, but not the type of use this is aiming for isn't it?